### PR TITLE
Update release notes july 2019

### PR DESCRIPTION
**Pull request by eddietejeda**
This pull request was previously here https://github.com/18F/cg-site/pull/1446

But we moved from fork to branch.

### CODE REVIEW

**Review by siennathesane — DISMISSED** [review]

File: content/updates/2019-07-12-VPN-backhaul-drupal-8-and-new-team-members.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+## Removed
+
+### Removed: CNAME ALIAS method for verifying DNS ownership
+
+When using the [CDN service](https://cloud.gov/docs/services/cdn-route/) to set up a custom domain name for an application, cloud.gov now only offers certificate provisioning via DNS challenges. We stopped supporting the CNAME ALIAS method. See the [updated steps for setting up DNS](https://cloud.gov/docs/services/cdn-route/#how-to-set-up-dns).
+
+### Removed: Experimental NFS (Network File System) volume service
+
+We are no longer offering our experimental Network File System (NFS) volume service. We are exploring alternatives to persistent filesystem volumes to support persistant storage.
```
Comment (hillaryj) [inline content/updates/2019-07-12-VPN-backhaul-drupal-8-and-new-team-members.md]: The second sentence should probably be "We are exploring alternative persistent filesystem volumes..."

Comment (eddietejeda) [reply content/updates/2019-07-12-VPN-backhaul-drupal-8-and-new-team-members.md]: good catch

Comment (siennathesane) [reply content/updates/2019-07-12-VPN-backhaul-drupal-8-and-new-team-members.md]: We aren't exploring persistent filesystem volumes for CF for the foreseeable future.

**Review by siennathesane — APPROVED** [review]
LGTM

**Pull request merged**